zoo/alexnet.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2021 Loro Francesco
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__author__ = "Francesco Loro"
+__email__ = "francesco.official@gmail.com"
+__supervisor__ = "Danilo Pau"
+__email__ = "danilo.pau@st.com"
+
+# Download pretrained weight from:
+# Alexnet -> https://drive.google.com/file/d/1-65sB1xnJuOoPhL00TYY0s3Fov0zxBHJ/view?usp=sharing
+
+import qkeras as q
+import tensorflow as tf
+import larq as lq
+from utils import compare_network, create_random_dataset, dump_network_to_json
+
+# Define path to the pre-trained weights
+PATH_ALEXNET = "weights/binary_alexnet_weights.h5"
+ALEXNET_NAME = "alexNet"
+
+
+class AlexNet:
+  """
+  Class to create and load weights of: alexnet
+  Attributes:
+        network_name: Name of the network
```

Comment:
one sentence in the first line of the docstring followed by one empty line

ref: https://google.github.io/styleguide/pyguide.html#384-classes

could you use 2 or 4 spaces for the indentation? thanks